benedict/serializers/ini.py

```diff
@@ -3,7 +3,7 @@
 from __future__ import absolute_import
 
 import io
-from typing import Dict
+from copy import deepcopy
```

Comment:
Instead of using `copy.deepcopy`, you could use `benedict.core.clone` method, which is a `deepcopy` wrapper, in this way there will be a single entry point for all `deepcopy` calls.